sync_check_terraform/modules/sync_check/service/init.sh

```diff
@@ -10,3 +10,26 @@ dnf clean all
 gem install slack-ruby-client sys-filesystem
 
 nohup /bin/bash ./run_service.sh > run_service_log.txt &
+
```

Comment:
All of this looks exactly the same as `daily_snapshot_terraform/modules/daily_snapshot/service/init.sh`. Which begs a question: should we have one template instead of these two files?